### PR TITLE
Options description of toLocaleString()

### LINKED ISSUE #11174: Issue with "Number.prototype.toLocaleString()": needs description of all the options
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

#### What information was incorrect, unhelpful, or **incomplete**?
Options: 

#### Specific section or headline?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#using_options

#### What did you expect to see?
There's no description of all the options. I found a list in German language, but I don't know if that's the correct list.

#### Did you test this? If so, how?
no.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/number/tolocalestring`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md
* Last commit: https://github.com/mdn/content/commit/0347d27d3ba30907174acfd3a49976a7ac19ca2b
* Document last modified: 2021-09-17T15:25:55.000Z

</details>

### PR DESCRIPTION
**Pull request by SaurabhCha**
Issue with "Number.prototype.toLocaleString()": needs description of all the options #11174
Issue: There's no description of all the options.
Fixes https://github.com/mdn/content/issues/11174

Changes: I have added all the options which are available for it.

#### Summary

Add available options after research and trying everything myself.

#### Motivation

It will allow users to see all the possible option values which weren't available earlier.

#### Metadata

- [ ] Adds a new document
- [ x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by SaurabhCha** [comment]
@teoli2003 Please review the PR.
And also please let me know if anything else is required. Thanks.

**Review by wbamberg — DISMISSED** [review]
Thanks for the PR, @SaurabhCha ! I think this should appear under the "Parameters" heading of the "Syntax" section, as they do for the constructor page: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#parameters .

**Comment by SaurabhCha** [comment]
@wbamberg Thank you for letting me know! I will make the changes.

**Comment by SaurabhCha** [comment]
Hi @wbamberg, I have made the required changes. Please let me know if anything else is required. Thanks!

File: files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 1]
```diff
@@ -127,7 +196,6 @@ console.log(number.toLocaleString(['ban', 'id']));
 ```
 
 ### Using `options`
-
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md:130]: The empty line here is necessary. Can you re-add it?

File: files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 2]
```diff
@@ -32,6 +32,75 @@ should be used. In implementations, which ignore the `locales` and
 `options` arguments, the locale used and the form of the string returned are
 entirely implementation dependent.
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md:33]: I would add an entry for `locales` like `options`:
```suggestion
entirely implementation-dependent.

- `locales` {{optional_inline}}
  - : A string containing the {{Glossary("locale")}} to convert too.
```

File: files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 3]
```diff
@@ -32,6 +32,75 @@ should be used. In implementations, which ignore the `locales` and
 `options` arguments, the locale used and the form of the string returned are
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md:32]: On line 29 above replace _arguments_ by _parameters_.

File: files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 4]
```diff
@@ -127,7 +196,6 @@ console.log(number.toLocaleString(['ban', 'id']));
 ```
 
 ### Using `options`
-
 The results provided by `toLocaleString` can be customized using the
 `options` argument:
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md:200]: We use parameter (and no more argument)
```suggestion
`options` parameter:
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
This is really good and almost ready to be merged. I've added a few minor requests. Sorry for the delay.

File: files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md (hunk 1 above)
Comment (SaurabhCha) [reply files/en-us/web/javascript/reference/global_objects/number/tolocalestring/index.md:130]: Added. Thanks

**Comment by SaurabhCha** [comment]
@teoli2003 I have made the changes. Please let me know if I missed anything or other changes are required. Thanks.

**Review by teoli2003 — APPROVED** [review]

**Comment by SaurabhCha** [comment]
Hi @wbamberg, are any changes required?

**Review by teoli2003 — APPROVED** [review]
Let's merge this. Any more changes, if needed, can be done in follow-up PRs.

**Comment by teoli2003** [comment]
Overriding the test failure as this is not touching redirects at all.

Thanks a lot!

**Comment by wbamberg** [comment]
Sorry to be slow reviewing this. This PR doesn't format the Markdown DL correctly:

Screen Shot 2022-06-10 at 10 59 17 PM

...so will need a follow-up.

**Comment by wbamberg** [comment]
Never mind, it looks like https://github.com/mdn/content/pull/17176 fixed it already. :)

**Pull request merged**